qa/L0_sequence_batcher/bool_type_sequence_models/graphdef_sequence_bool/config.pbtxt

```diff
@@ -0,0 +1,45 @@
+name: "graphdef_sequence_bool"
```

Comment:
I forgot to mention this in the previous round of review: we generate the models used for `L0_sequence_batcher` with script (qa/common/gen_qa_sequence_models.py), and you should be able to generate boolean sequence model as well with minor modifications on the gen script. Then you would want to upload the new models to our test model directory.

And then with the comment regarding test script, the boolean case should be covered without the change in `test.sh`

Reply:
I see, thanks for the comments, will modify.

Reply:
You can remove the `bool_type_sequence_models` folder as now it is not longer needed.